dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/Components/GeneratorWithControl/Controls/Governor.mo

```diff
@@ -0,0 +1,146 @@
+within Dynawo.Examples.Nordic.Components.GeneratorWithControl.Controls;
```

Comment:
Same remarks as in ExciterAvrPssOel regarding the comments for parameters, variables, indentation, blank lines, copyright, dynawo types, etc.

Reply:
resolved, see [e1000c7](https://github.com/dynawo/dynawo/pull/2394/commits/e1000c7adff25e271a2e2214bb7197d005b00633)